_maps/map_files/cyberiad/cyberiad.dmm

```diff
@@ -46238,7 +46238,6 @@
 	dir = 2;
 	network = list("Research","SS13")
 	},
-/obj/structure/reagent_dispensers/oil,
```

Comment:
*hopefully* this can be merged after the mapping queue is dealt with. I hope so atleast.

Reply:
Oh right, map changes. RIP this PR for a while then.

Reply:
If only there was another use for oil so the oil tanks could stay in...

Reply:
Well cyborgs could probably use it seeing that they actually have oil spatter out of them when they are brute damaged. That could be another form of repair and/or maintenance for them, refilling oil. Less oil means slower operation and movement

Reply:
have robotics hand the tank to chemistry *shrugs